### PR TITLE
RCB-616: update python Jenkins script

### PR DESCRIPTION
**Pull request by yalcantara**
This PR is needed to update the reference for the python examples in Dockerhub

### CODE REVIEW

**Review by matt-l-01 — COMMENTED** [review]
Everything looks good from my end...I am going to get a second opinion on this before approval because this is my first verification and a larger verify!

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (seth-basistech) [inline .vscode/launch.json:1]: I think this file can be deleted.  You can probably add `.vscode` to `.gitignore` too.

File: CI.Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    sh "docker run \
+            --pull always \
+            --rm --volume ${sourceDir}:/source \
+            python:${ver}-slim \
+            bash -c \"apt-get update && \
+            apt-get install -y wget unzip && \
+            pip3 install tox && \
+            cd /root/ && \
+            wget -q https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.8.0.2856-linux.zip && \
```
Comment (seth-basistech) [inline CI.Jenkinsfile:23]: Leave a TODO comment somewhere in here to pull out the Sonar run to a single Python version (the latest?).  We only need to download the archive and execute it once.  Not for each Python version.

File: CI.Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
+
+
+def versions = [3.11, 3.10, 3.9, 3.8, 3.7]
```
Comment (seth-basistech) [inline CI.Jenkinsfile:3]: Can you update the `classifiers` in `setup.py` to match this list.

**Review by seth-basistech — CHANGES_REQUESTED** [review]
This PR should include the removal of the deprecated code too so that it is ready to go in the next release cycle.

File: CI.Jenkinsfile (hunk 2 above)
Comment (yalcantara) [reply CI.Jenkinsfile:25]: TODO comment added!

File: CI.Jenkinsfile (hunk 3 above)
Comment (yalcantara) [reply CI.Jenkinsfile:3]: Updated!

**Pull request merged**